lesson_3/js/index_1.js

```diff
@@ -0,0 +1,9 @@
+'use strict'
+
+let a = 1;
+let b = 2;
+
+b = b - a;
+a = a + b;
+
+console.log(a, b)
```

Comment:
have you tested `a, b` variables with other different values? This is incorrect logic